### PR TITLE
Converted hedging to a TriggeredAttachment

### PR DESCRIPTION
**Pull request by bsharpataz**
... to help it go through to the jsonld.  Format is now trigger is the
num of hedging words (as a string) and the quantifiers holds the list of
the hedging terms

@kwalcock if there is a cleaner solution please feel free to modify however you want!

@bgyori I verified, and with this branch hedging gets through to the jsonld.  for example, for the sentence: `Rainfall might cause increased flooding.` we get:
```...
{
    "@type" : "Extraction",
    "@id" : "_:Extraction_1",
    "type" : "relation",
    "subtype" : "causation",
    "labels" : [ "Causal", "DirectedRelation", "EntityLinker", "Event" ],
    "text" : "Rainfall might cause increased flooding",
    "rule" : "ported_syntax_1_verb-Causal",
    "canonicalName" : "rainfall cause increase flooding",
    "provenance" : [ {
      "@type" : "Provenance",
      "document" : {
        "@id" : "_:Document_1"
      },
      "documentCharInterval" : [ {
        "@type" : "Interval",
        "start" : 1,
        "end" : 39
      } ],
      "sentence" : {
        "@id" : "_:Sentence_1"
      },
      "positions" : [ {
        "@type" : "Interval",
        "start" : 1,
        "end" : 5
      } ]
    } ],
    "states" : [ {
      "@type" : "State",
      "type" : "HEDGE",
      "text" : "1",
      "modifiers" : [ {
        "@type" : "Modifier",
        "text" : "might"
      } ]
    } ],
...
```

### CODE REVIEW

**Comment by bsharpataz** [comment]
I think this closes #444

**Comment by kwalcock** [comment]
I need to document this on the JSON-LD page.  I'm a little concerned about text: "1" and how that compares to other usages.  I can't tell from the documentation what we were thinking there.  Merge shortly.

**Comment by MihaiSurdeanu** [comment]
@bsharpataz: can you please clarify the JSON format?

**Comment by kwalcock** [comment]
I added documentation on the JSON-LD page.  It seems that by analogy to INC, DEC, QUANT, the first text in the helpful! example would be "might" rather than "1" and modifiers would be for other words as in "definitely might".  It's probably a side-effect of some implementation detail I haven't checked on, but for INC we have provenance.  Is it not available here?  I probably need a memory refresh.

**Comment by bsharpataz** [comment]
> I added documentation on the JSON-LD page. It seems that by analogy to INC, DEC, QUANT, the first text in the helpful! example would be "might" rather than "1" and modifiers would be for other words as in "definitely might". It's probably a side-effect of some implementation detail I haven't checked on, but for INC we have provenance. Is it not available here? I probably need a memory refresh.

Yes -- by analogy that's what it would be, but I was thinking that in terms of what the downstream users need, they really just need a number to convey how hedged it is.  I guess it could also be just like the other (INC, DEC, etc), and the assemblers could count how many hedging attachments there are...?  Thoughts?

**Comment by MihaiSurdeanu** [comment]
I think we should stick with a number, because that number is theoretically unbounded. It would be hard to come with verbose descriptions for all.

**Comment by bsharpataz** [comment]
> I think we should stick with a number, because that number is theoretically unbounded. It would be hard to come with verbose descriptions for all.

I think we may go with the standard TriggeredAttachment like Inc/Dec and ask the assemblers to count how many Hedge attachments there are (or rather, look for the presence of a Hedge attachment).  This would port well to negation too (i.e., look for existence of a Negation)

**Comment by bsharpataz** [comment]
@bgyori @MihaiSurdeanu @kwalcock 
updated main comment to reflect current design.  
TLDR you can look for presence of HEDGE on a Causal relation or count them, however you want!
I plant to do negation similarly (except for the counting, i.e. just presence/absence)

**Comment by bgyori** [comment]
Sure, this looks good to me! Once this PR and negations are merged, I'll update INDRA to extract both.

**Comment by kwalcock** [comment]
@bsharpataz, did you notice the failing tests?  It doesn't look like a Travis problem.
